### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Ocunidee**
- typescript compiler is now in strict mode
- stricter linter rules are in place

The purpose of this PR is to clean the code before new features are added

### CODE REVIEW

**Comment by tetram** [comment]
Should use an editorconfig.

File: src/app/components/app/app.component.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import { Component, OnInit } from '@angular/core';
-import { Router, NavigationStart, NavigationEnd, Event as NavigationEvent } from '@angular/router';
-import { LoginService } from '../../services/login/login.service';
-import { Location, PopStateEvent } from '@angular/common';
+import { Component, OnInit } from '@angular/core'
+import { Router, NavigationStart, NavigationEnd, Event as NavigationEvent } from '@angular/router'
+import { LoginService } from '../../services/login/login.service'
+import { Location, PopStateEvent } from '@angular/common'
+import { takeUntil } from 'rxjs/operators'
```
Comment (bcoulaud) [inline src/app/components/app/app.component.ts]: Unused import

File: src/app/services/issue-type/issue-types.service.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+import { HttpHeaders, HttpClient } from '@angular/common/http'
+import { Observable } from 'rxjs'
+import { environment } from 'src/environments/environment'
 
-const headers = new HttpHeaders({'Content-Type':  'application/json'});
+const headers = new HttpHeaders({'Content-Type':  'application/json'})
 
 const httpOptions = {
-  headers: headers,
+  headers,
```
Comment (bcoulaud) [inline src/app/services/issue-type/issue-types.service.ts]: Are you sure ?

File: src/app/services/kanban-issues/kanban-issues.service.ts [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+import { HttpClient, HttpHeaders } from '@angular/common/http'
+import { environment } from '../../../environments/environment'
+import { KanbanColumn } from './kanban-column'
 
-const headers = new HttpHeaders({'Content-Type':  'application/json'});
+const headers = new HttpHeaders({'Content-Type':  'application/json'})
 
 const httpOptions = {
-  headers: headers,
+  headers,
```
Comment (bcoulaud) [inline src/app/services/kanban-issues/kanban-issues.service.ts]: Header not set

File: src/app/services/printing-configuration/printing-configuration.service.ts [hunk 4]
```diff
@@ -1,80 +1,80 @@
...
+import { HttpClient, HttpHeaders } from '@angular/common/http'
+import { environment } from 'src/environments/environment'
+import { Configuration } from './configuration'
 
-const headers = new HttpHeaders({'Content-Type':  'application/json'});
+const headers = new HttpHeaders({'Content-Type':  'application/json'})
 
 const httpOptions = {
-  headers: headers,
+  headers,
```
Comment (bcoulaud) [inline src/app/services/printing-configuration/printing-configuration.service.ts]: Header not set

File: src/app/services/login/login.service.ts [hunk 5]
```diff
@@ -1,47 +1,53 @@
...
+import { HttpClient, HttpHeaders } from '@angular/common/http'
+import { environment } from '../../../environments/environment'
+import { Board } from './board'
 
-const headers = new HttpHeaders({'Content-Type':  'application/json'});
+const headers = new HttpHeaders({'Content-Type':  'application/json'})
 
 const httpOptions = {
-  headers: headers,
+  headers,
```
Comment (bcoulaud) [inline src/app/services/login/login.service.ts]: Header missing

File: src/app/services/sprints/sprint.service.ts [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
+import { Observable } from 'rxjs'
+import { HttpClient, HttpHeaders } from '@angular/common/http'
+import { environment } from '../../../environments/environment'
 
-var headers = new HttpHeaders({'Content-Type':  'application/json'});
+const headers = new HttpHeaders({'Content-Type':  'application/json'})
 
 const httpOptions = {
-  headers: headers,
+  headers,
```
Comment (bcoulaud) [inline src/app/services/sprints/sprint.service.ts]: missing header

File: src/app/services/kanban-issues/kanban-issues.service.ts [hunk 7]
```diff
@@ -1,15 +1,15 @@
-import { Injectable } from '@angular/core';
-import { Observable } from 'rxjs';
-import { HttpClient, HttpHeaders } from '@angular/common/http';
-import { environment } from '../../../environments/environment';
-import { KanbanColumn } from './kanban-column';
+import { Injectable } from '@angular/core'
+import { Observable } from 'rxjs'
+import { HttpClient, HttpHeaders } from '@angular/common/http'
+import { environment } from '../../../environments/environment'
```
Comment (cihancnr) [inline src/app/services/kanban-issues/kanban-issues.service.ts]: import { environment } from 'environments/environment';

**Pull request merged**